### ISSUE TITLE
Add SPDX License Identifier and Copyright Statement to relevant files

### ISSUE THREAD

**Issue by adityasaky** [in-toto/in-toto#439]
**Description of issue or feature request**:
The [Badge App](https://bestpractices.coreinfrastructure.org/en/projects/1523?criteria_level=2#basics) for CII Best Practices's Gold level requires each source file to contain a copyright statement and an SPDX License Identifier.

**Comment by lukpueh** [comment]
This is [what this looks like for tuf](https://github.com/theupdateframework/tuf/blob/1d234798d6214a02301a97906b0f86a769a1b2fa/tuf/api/metadata.py#L1-L2):

```
# Copyright New York University and the TUF contributors
# SPDX-License-Identifier: MIT OR Apache-2.0
```

It think for in-toto it will be.

```
# Copyright New York University and the in-toto contributors
# SPDX-License-Identifier: Apache-2.0
```

Note that we [used to list year ranges](https://github.com/theupdateframework/tuf/blob/1d234798d6214a02301a97906b0f86a769a1b2fa/tuf/repository_tool.py#L4-L5) in tuf inline copy right statements, but gave it up because its a nuisance to maintain (see https://github.com/secure-systems-lab/sample-documents/pull/5) and probably does not make a huge difference (cc @jku).

I think @MVrachev from the TUF team wanted to add a recommendation to our shared style guide.

**Comment by adityasaky** [comment]
Closed by #440.
